llm_memory/components/embedding_provider.py

```
@@ -810,6 +824,50 @@ async def _get_embeddings_with_batch(
+    def _expand_aggregated_embedding(
+        self, embeddings: List[List[float]], texts: List[str]
+    ) -> List[List[float]]:
+        """
+        兼容上游对多输入返回单个聚合向量的行为。
+
+        上游 provider 仍负责请求与解析；这里仅把符合结构特征的聚合结果
+        扩展为本批次所需的向量数量，避免后续 FAISS 回填因数量不一致失败。
+        """
+        if self._is_aggregated_embedding_result(embeddings, texts):
+            self.logger.warning(
+                "嵌入提供商返回聚合向量，已扩展到批次内所有文本: "
```

Ruff (0.15.15)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)